com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -6,11 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [13.1.5] - 2021-12-17
 
+### Changed
+- DecalRendererFeature is now using RTHandles internally.
```

Comment:
The changed category is only used for changing in user facing aspects. This change doesn't need to specific this. The reason I mention is that RM checks the changelog to see if we follow semver, and in this case this would imply we change on a patch package which is not true. This is purely bug fix. For sake of clarity we should remove this.